app/routes/github.js

```diff
@@ -8,17 +8,29 @@ const {
 } = Ember;
 
 const CODE_INVALID = 'The GitHub authorization code is invalid.';
+const STATE_INVALID = 'Something went wrong in the connect process. Please try again';
 
 export default Route.extend(AuthenticatedRouteMixin, {
-  queryParams: { code: { refreshModel: true } },
+  queryParams: {
+    code: { refreshModel: true },
+    state: { refreshModel: true }
```

Comment:
These could use some inline commenting explaining why we use `refreshModel` here.